source/_integrations/unifiprotect.markdown

```diff
@@ -213,6 +213,14 @@ Your main UniFi Protect NVR device also gets a number of diagnostics sensors tha
 * **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as a state attribute though to help map sensor to disk.
 * **Utilization and Storage Sensors**: Several other sensors are also added for uptime, hardware utilization, and distribution details of the video on disk.
 
+## Ignoring Devices
+
+If for some reason you want to not have a UniFi Protect device appear in Home Assistant, you can simply "delete" the device from Home Assistant.
```

Comment:
Let's avoid terms like "simply", what might be simple for you, might not be simply something someone else would do. It might be received as a little condescending by the reader.

```suggestion
If, for some reason, you do not want to have an UniFi Protect device appear in Home Assistant, you can "delete" the device from Home Assistant.
```